Zend/zend_max_execution_timer.h

```diff
@@ -21,10 +21,12 @@
 
 #include "zend_long.h"
 
+BEGIN_EXTERN_C()
 /* Must be called after calls to fork() */
 ZEND_API void zend_max_execution_timer_init(void);
```

Comment:
Only this function must be called in extensions if they `fork()`. It nay not be necessary to protect the two other definitions.